package.json

```diff
@@ -2,62 +2,42 @@
   "name": "forklift-ui",
   "version": "0.0.1",
   "description": "OpenShift VM Migration UI",
-  "main": "deploy/server.js",
-  "repository": "https://github.com/konveyor/forklift-ui.git",
-  "license": "Apache",
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/konveyor/forklift-ui.git"
+  },
+  "main": "pkg/api/src/server.js",
   "private": true,
   "scripts": {
     "dr:surge": "node dr-surge.js",
+    "setup:dev:remote": "node ./pkg/api/scripts/remote-dev.js",
     "build": "npm run clean && NODE_ENV=production $(npm bin)/webpack --config ./config/webpack.prod.js",
     "build:remote": "DATA_SOURCE=remote npm run build",
     "build:mock": "DATA_SOURCE=mock FORKLIFT_UI_GIT_COMMIT=$(git rev-parse --short HEAD) npm run build && npm run dr:surge",
-    "setup:dev:remote": "node --trace-deprecation ./scripts/remote-dev.js",
-    "start": "node --trace-deprecation deploy/server.js",
+    "start": "node ./pkg/api/src/server.js",
```

Comment:
Instead of directly running the server file here, should we use `npm run start -w api`?

Reply:
This honestly has me wondering if we should rename the `api` package to `server` because in prod it isn't just the API, it hosts the whole UI

Reply:
IMO `api` still makes sense, even though it's mostly just serving the UI over a couple routes, it still stands as an api layer of its own, has multiple api routes defined, likely be extended in future, etc.

Just my two cent :)

Reply:
You got me divide her ;)
Well API is a service and serving the page isn't a form of api. :)
Let's leave it for now?

Reply:
Regarding the start script, it makes sense to have the run attached to it's package